frontend/src/app/main/ui/workspace/sidebar/assets.cljs

```diff
@@ -571,7 +571,10 @@
         [:*
           [:span (:name file)]
           [:span.tool-link.tooltip.tooltip-left {:alt "Open library file"}
-           [:a {:href (str "#" url) :target "_blank"} i/chain]]])]
+           [:a {:href (str "#" url)
+                :target "_blank"
+                :on-click (fn [event] (dom/stop-propagation event))}
```

Comment:
In this case you don't need to create an aditional anonymous funcion, just pass `:on-click dom/stop-propagation`

Reply:
True! Changed.